SeriesMaster-Backend/lib/showsDB.js

```diff
@@ -0,0 +1,21 @@
+async function postShow(database, show) {
+  const collection = database.collection("shows");
+  await collection.insertOne(show);
+}
+
+async function findShow(database, showId) {
+  const collection = database.collection("shows");
+  const show = await collection.findOne({ id: showId });
+
+  return show;
+}
+async function findShows(database, showsId) {
+  const collection = database.collection("shows");
+  const shows = await collection.find({ id: { $in: showsId } }).toArray();
+
+  return shows;
+}
+
+exports.postShow = postShow;
+exports.findShow = findShow;
+exports.findShows = findShows;
```

Comment:
Alternatively you can export it like this:

module.exports = {
postShow,
findShow,
findShows
}
And then import it like:

const {postShow, ... } = require 'path/to/file'

Reply:
Thank you!